src/main.rs

```diff
@@ -1,16 +1,104 @@
-use anyhow::{ Result};
-use cli_lang_tool::{Dictionary, Word};
+use anyhow::{Result};
+use clli::{Dictionary, Word};
 use std::{fs::File, io::{prelude::*, BufReader}, path::PathBuf};
 use serde::{Serialize, Deserialize};
 use rand::Rng;
-use structopt::StructOpt;
+use clap::{ Parser, Subcommand};
 
 
 const FILE_PATH: &str = "./examples/storage/turk.json";
```

Comment:
Path could be specified with a flag. Such as:
```
clli random --path examples/storage/turk.json
```

Reply:
ok